bin/admin/install-solvcon.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+if [ $# -lt 1 ]
+then
+    echo "Usage : $0 <solvcon env configuration file>"
+    exit
+fi
+solvcon_env_file=$1
+git clone https://github.com/solvcon/solvcon.git
+
+# you never know the env is ready to use SOLVCON or not, so
+# enable the env anyway.
+source $solvcon_env_file
+# basic env
+export SCSRC=$HOME/solvcon
+# prepare the ssh configuration to test rpc features on the localhost
```

Comment:
This is better to me. I will prepare ``gce-ssh-keygen.sh``. Besides, ``install-solvcon.sh`` will invoke it. Let me think about how to implement ``gce-ssh-keygen.sh`` to meet the requrement of the bullet 2 and 3. The first thing comes to my mind is how to propagate the public key between nodes. Any idea?

For example, ``gce-ssh-keygen.sh`` may create a ftp instance used to exchange our own temporary public SSH key between nodes. Maybe I should have a look how the others exchange their public keys between their nodes.